src/Bundle/CoverallsBundle/Repository/JobsRepository.php

```diff
@@ -175,7 +182,7 @@ protected function send()
                 $this->logResponse($response);
             }
 
-            return;
+            return true;
```

Comment:
sadly, we can't stop catching errors here and do it one level higher, and log also there - as it would be bc breaker... for that, returning poor men's bool result